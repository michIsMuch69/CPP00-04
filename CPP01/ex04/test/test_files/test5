Thi$ i$ a te$$t
Thi$ i$ a te$$t
Thi$ i$ a te$$t
        Thi$ i$ a te$$t
        Thi$ i$ a te$$t
    Thi$ i$ a te$$t
    Thi$ i$ a te$$t
Thi$ i$ a te$$t
Thi$ i$ a te$$t
Thi$ i$ a te$$t









Thi$ i$ a te$$t
