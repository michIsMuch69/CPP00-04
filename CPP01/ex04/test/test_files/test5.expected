Thizz izz a tezzzzt
Thizz izz a tezzzzt
Thizz izz a tezzzzt
        Thizz izz a tezzzzt
        Thizz izz a tezzzzt
    Thizz izz a tezzzzt
    Thizz izz a tezzzzt
Thizz izz a tezzzzt
Thizz izz a tezzzzt
Thizz izz a tezzzzt









Thizz izz a tezzzzt
